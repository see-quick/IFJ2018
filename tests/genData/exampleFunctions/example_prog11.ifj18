# Draw a pyramide

print "Enter number of rows of pyramide \n"
rows = inputi

print "\n\n"

i = 1
j = 1

while i <= rows do
  j = 1
  while j <= i do

      print j
      j = j + 1
  end
  print "\n"
  i = i + 1
end
