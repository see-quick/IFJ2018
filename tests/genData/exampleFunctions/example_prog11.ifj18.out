Enter number of rows of pyramide 
5

1
12
123
1234
12345
